chains/orchestrator-relays/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -538,43 +403,6 @@ mod test_helpers {
     }
 }
 
-/// Rewards the relayer that processed a native token transfer message
```

Comment:
this might conflict with @Agusrodri PR https://github.com/moondance-labs/tanssi/pull/1105

Reply:
Merge conflict should be fixed :)